endpoints/openrtb2/auction.go

```diff
@@ -1567,3 +1584,30 @@ func getAccountID(pub *openrtb2.Publisher) string {
 	}
 	return metrics.PublisherUnknown
 }
+
+func storedRequestErrorChecker(err error, requestJson []byte, storedRequests map[string]json.RawMessage, storedBidRequestId string) ([]byte, []error) {
+	if err != nil {
```

Comment:
The original `err` passed in isn't used other than for this check, which is already covered on both code paths to this method. Consider removing the argument.